src/components/partner/partner.service.ts

```diff
@@ -29,6 +35,7 @@ export class PartnerService {
   constructor(
     @Logger('partner:service') private readonly logger: ILogger,
     @Inject(forwardRef(() => AuthorizationService))
+    private readonly projectService: ProjectService,
```

Comment:
You split the Inject decorator and the authorizationService lines here. So it's injecting AuthorizationService for the projectService property.

Reply:
Ugh.  Thanks.